core/src/main/java/io/grpc/ServerCallHandler.java

```diff
@@ -39,7 +39,7 @@
  */
 @ExperimentalApi("https://github.com/grpc/grpc-java/issues/933")
 @ThreadSafe
-public interface ServerCallHandler<RequestT, ResponseT> {
+public abstract class ServerCallHandler<RequestT, ResponseT> {
```

Comment:
I see this as a single _method_. Using an abstract class turns it into more of an object. And the interface fits very well with Java 8's lambdas (if we were so inclined) which are for single methods. I also don't think we should be adding things here, but if we really need it we can have a `ServerCallHandler2` and auto-adapt.